pyproject.toml

```diff
@@ -22,11 +22,11 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7.13"
+python = "^3.7"
 requests = "^2.23"
-cachetools = "^4.1"
+cachetools = "^5.3"
 rsa = "^4.0"
-importlib-metadata = {version = "^3.6", python = "<3.8"}
+importlib-metadata = "^6.4"
```

Comment:
Is there any reason for removing `python = "<3.8"`?

Reply:
oh, I see. It makes sense.
To use `importlib-metadata` in all versions, I guess we need to remove these lines:

https://github.com/mdsol/mauth-client-python/blob/f08280d521c4b1bbf270d58200f342daa2cc8777/mauth_client/__init__.py#L2-L5

Reply:
To be fair I don't understand what the library does or why it's needed. Potentially we can get away with loosing the version specification to set a minimum version if we don't need features provided in newer versions.

Reply:
I don't think we need new features. This dependency can be removed when we drop python `3.7` support.